plugins/tech-radar/src/index.ts

```diff
@@ -26,6 +26,9 @@ export {
   TechRadarPage,
 } from './plugin';
 
+/**
+ * @deprecated Use plugin extensions instead
+ */
 export { RadarPage as Router } from './components/RadarPage';
```

Comment:
This combo should help get `@deprecated` in the right place without breaking API

```ts
import { RadarPage } from './components/RadarPage'

/** @deprecated */
export const Router = RadarPage
```